### PR TITLE
Improved default Layout

### PR DESCRIPTION
**Pull request by froooze**
<h2>General</h2>
Closes #[Insert issue number]

[Insert description of what you did, any description how to test or check your changes are welcome]

<h2>General</h2>
Please make sure the following is done:

- Pull request is onto develop
- If you haven't already, have a look at
  - https://github.com/bitshares/bitshares-ui/blob/develop/CODE_OF_CONDUCT.md
  - https://github.com/bitshares/bitshares-ui/blob/develop/CONTRIBUTING.md

<h2>Code Preparation</h2>

_Please review all your changes one last time before committing_

- [x] Check for unused code
- [x] No unrelated changes are included
- [x] None of the changed files are reformatting only
- [x] Code is self explanatory or documented
- [ ] All written text is properly translated (english language)

<h2>Testing</h2>

_The branch has been tested on the following browsers (desktop and mobile view)_

- [x] Chrome 
- [ ] Opera
- [x] Firefox
- [ ] Safari

<h2>User interface changes</h2>

_Delete this section if there weren't any UI changes. Please make sure you tested your changes in all themes_

- [x] Dark
- [ ] Light
- [x] Midnight

_Please provide screenshots/licecap of your changes below_

### CODE REVIEW

File: app/components/Exchange/Exchange.jsx [hunk 1]
```diff
@@ -361,12 +361,12 @@ class Exchange extends React.Component {
...
                 my_history: 1,
                 history: 1,
-                my_orders: 2,
+                my_orders: 1,
                 open_settlement: 2
             }),
             panelTabsActive: ws.get("panelTabsActive", {
                 1: "my_history",
-                2: "my_orders"
+                1: "my_orders"
```
Comment (sschiessl-bcp) [inline app/components/Exchange/Exchange.jsx]: What do you want to achieve here?

Comment (froooze) [reply app/components/Exchange/Exchange.jsx]: image

To have my trades and my open orders next to market trades.

**Comment by abitmore** [comment]
FWIW as mentioned in the issue, I don't think it's the solution.

**Comment by sschiessl-bcp** [comment]
Only layout change, subjective

**Pull request closed without merging**